tmt/steps/prepare/__init__.py

```diff
@@ -1,9 +1,38 @@
 # coding: utf-8
 
-""" Prepare Step Classes """
+""" Prepare Step Class """
 
 import tmt
+import os
+import shutil
+import subprocess
+from tmt.steps.Prepare import shell, beakerlib
 
 
 class Prepare(tmt.steps.Step):
-    """ Configure environment for testing (e.g. ansible playbook) """
+    name = 'prepare'
+
+    def __init__(self, data, plan):
+        """ Initialize the Prepare step """
+        self.super = super(Prepare, self)
+        self.super.__init__(data, plan)
+
+    def wake(self):
+        """ Wake up the step (process workdir and command line) """
+        self.super.wake()
+        self.set_default('how', 'shell')
+
+    def show(self):
+        """ Show discover details """
+        self.super.show(keys = ['how', 'script'])
+
+    def go(self):
+        """ Prepare the test step """
+        self.super.go()
+        self.plan.provision.prepare(self.data['how'], self.data['script'])
+
+        # TODO: EXECUTE:
+        #   => self.plan.provision.execute("nohup", "run.sh", "..", ">generalout", "2>generalerr, "&"):
```

Comment:
Oops, this should probably go to  `steps/__init__.py` (_well, it's just a suggestion_).

Reply:
@pvalena  why would we run.sh in prepare??

Reply:
That was just a note for me to do that in execute. Please disregard that, as I've already PRs on the way wich fixes this, and implemets it on proper place.